### DISCUSSION TITLE
Could not push change to the repository

### DISCUSSION THREAD

**Discussion by dngduy132** [WeblateOrg/weblate#12985] [Q&A]
Previously, we migrated storage, and during this process, the person handling it accidentally deleted the Postgres Stateful Set. We then redeployed it with its current PVC. However, afterward, I encountered an alert saying, "Could not push the repository" with the following error log:

gunicorn stderr | weblate.vcs.base.RepositoryError: fatal: detected dubious ownership in repository at '/app/data/vcs/ipension-translation-de/glossary'
gunicorn stderr | To add an exception for this directory, call:
gunicorn stderr |
gunicorn stderr | git config --global --add safe.directory /app/data/vcs/ipension-translation-de/glossary

I tried running the git command directly on the Weblate pod, but it didn’t resolve the issue. Additionally, when I try to click "Repository maintenance," another error appears below. 

![image](https://github.com/user-attachments/assets/8212118b-5071-4923-84ad-6e0ddcf31fcb)

Could you help me fix this problem? Thanks for your time and assistance!

**Comment by nijel** [comment]
Probably the files are owned by different user than running Weblate.

**Reply by dngduy132** [reply]
Thanks @nijel for the answer, 
If the repository has become broken following the migration process, would it be possible to back up all existing data for the glossary component, then delete and re-import it into Weblate? Are there any potential risks with this approach?

**Reply by nijel** [reply]
It should work, but fixing the ownership should be easier to do, something like:

chown -R 1000:1000 /path/to/data-volume/
